molgenis-app/src/main/java/org/springframework/security/acls/jdbc/AclConfig.java

```diff
@@ -0,0 +1,104 @@
+package org.springframework.security.acls.jdbc;
+
+import org.molgenis.data.postgresql.DatabaseConfig;
+import org.molgenis.security.NoOpAuditLogger;
+import org.molgenis.security.acl.BitMaskPermissionGrantingStrategy;
+import org.molgenis.security.core.utils.SecurityUtils;
+import org.springframework.cache.Cache;
+import org.springframework.cache.support.NoOpCache;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.core.convert.support.DefaultConversionService;
+import org.springframework.security.acls.AclPermissionEvaluator;
+import org.springframework.security.acls.domain.AclAuthorizationStrategy;
+import org.springframework.security.acls.domain.AclAuthorizationStrategyImpl;
+import org.springframework.security.acls.domain.AuditLogger;
+import org.springframework.security.acls.domain.SpringCacheBasedAclCache;
+import org.springframework.security.acls.model.AclCache;
+import org.springframework.security.acls.model.MutableAclService;
+import org.springframework.security.acls.model.PermissionGrantingStrategy;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import javax.sql.DataSource;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * TODO move to org.molgenis sub-package once we upgraded to a spring-security-acl release with https://github.com/spring-projects/spring-security/issues/4814.
```

Comment:
resolve TODO

Reply:
Can't be resolved until a future spring-security-acl artifact release.